package-lock.json

```diff
@@ -0,0 +1,3997 @@
+{
+  "name": "solc-bin",
+  "version": "0.0.0",
+  "lockfileVersion": 1,
```

Comment:
Please do not include the lockfile in the PR. We don't want to lock to specific versions.